Remove selective filter tutorial (#3411)
viamrobotics/docs: docs/how-tos/deploy-ml.md

![Diagram of the camera component to data management service to ML model service to vision service pipeline.](/how-tos/ml-vision-diagram.png)

> You can use ML models to help your machines adapt their behavior to the world around them.

Reviewer: This is part of #3410 also?

Author: hadn't rebased yet

{{% expand "Just testing and want a dataset to get started with? Click here." %}}

We have two datasets you can use for testing, one with shapes and the other wise a wooden figure:

{{<imgproc src="/tutorials/data-management/shapes-dataset.png" resize="1200x" declaredimensions=true style="max-width:400px" alt="The shapes dataset." class="imgzoom fill aligncenter">}}

> {{< imgproc src="/tutorials/filtered-camera-module/viam-figure-dataset.png" style="max-width:400px" alt="The datasets subtab of the data tab in the Viam app, showing a custom 'viam-figure' dataset of 25 images, most containing the wooden Viam figure" class="imgzoom fill aligncenter" resize="1400x" >}}

Author: Note to self this should also get added to the datasets page

> 1. [Download the shapes dataset](https://storage.googleapis.com/docs-blog/dataset-shapes.zip) or [download the wooden figure dataset](https://storage.googleapis.com/docs-blog/dataset-figure.zip).
> 2. Unzip the download.
> 3. Open a terminal and go to the dataset folder.
> 4. In it you will find a Python script to upload the data to the Viam app.
> 5. Open the script and fill in the constants at the top of the file.
> 6. Run the script to upload the data into a dataset in Viam app:

Author: ```suggestion
1. [Download the shapes dataset](https://storage.googleapis.com/docs-blog/dataset-shapes.zip) or [download the wooden figure dataset](https://storage.googleapis.com/docs-blog/dataset-figure.zip).
1. Unzip the download.
1. Open a terminal and go to the dataset folder.
1. In it you will find a Python script to upload the data to the Viam app.
1. Open the script and fill in the constants at the top of the file.
1. Run the script to upload the data into a dataset in Viam app:
```

   ```sh {class="command-line" data-prompt="$" }
   python3 upload_data.py
   ```

> 7. Continue to [Train a machine learning model](#train-a-machine-learning-ml-model).

Author: ```suggestion
1. Continue to [Train a machine learning model](#train-a-machine-learning-ml-model).
```

{{% expand "Just testing and want a dataset to get started with? Click here." %}}

> We have two datasets you can use for testing, one with shapes and the other wise a wooden figure:

Reviewer: ```suggestion
We have two datasets you can use for testing, one with shapes and the other with a wooden figure:
```

{{<imgproc src="/services/ml/label.svg" class="fill alignleft" style="max-width: 250px" declaredimensions=true alt="Label data">}}
**2. Label your images**

Once you have enough images of the objects you'd like to identify captured and synced to the Viam app, use the interface on the [**DATA** tab](https://app.viam.com/data/view) to label your data.

You can label your images to create:

- **Detection models**: Draw **bounding boxes** around distinct objects within captured images.
  The trained model will enable your machine to be able to detect those objects on its own.

> - **Classification models**: Add **tags** to your images with class labels that describes it.

Reviewer: ```suggestion
- **Classification models**: Add **tags** to each of your images with class labels that describe it.
```

If the results exceed the confidence threshold, the **Run model** section shows a label and the responding confidence threshold.

When satisfied that your ML model is working well, continue to [deploy an ML model](#deploy-an-ml-model).

> If the vision service is not matching reliably, you will need to adjust your ML model by consider adding and labelling more images in your dataset.

Reviewer: ```suggestion
If the vision service is not detecting or classifying reliably, you will need to adjust your ML model by consider adding and labelling more images in your dataset.
```

> Ideally, you want your ML model to be able to identify objects with a high level of confidence, which is dependent on a robust source dataset.

Reviewer: ```suggestion
Ideally, you want your ML model to be able to identify objects with a high level of confidence, which is dependent on a robust training dataset.
```
Opt
